### DISCUSSION TITLE
How to choose frame rate?

### DISCUSSION THREAD

**Discussion by ghost** [deniscerri/ytdlnis#151] [Q&A]
I think the question is simple: how can I choose specific frame rate for a downloaded video (for example, 25 FPS)?

**Comment by deniscerri** [comment]
You can try to find a format that has that frame rate. Long press a format and it will tell you.

**Comment by prateekpatel1432** [comment]
To see tha side frame of fram because it's most important

**Comment by prateekpatel1432** [comment]
Fs87
